recipes/staden-io-lib/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "staden_io_lib" %}
+{% set version = "1.14.11" %}
+{% set hash = "a172cb66416794fdd9c1fc443f722f7e3439b52c99510b9a60f828392b9989e4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jkbonfield/io_lib/releases/download/io_lib-1-14-11/io_lib-1.14.11.tar.gz 
+  sha256: {{ hash }}
+
+build:
+    number: 0
+    skip: True  # [osx]
+
+
+requirements:
+    build:
+        - {{ compiler('c') }}
+    host:
+
+        - zlib
+        - xz
+        - curl
+        - libdeflate  
+  
+
+    run:    
+        - zlib
+        - xz
+        - curl
+        - libdeflate
+
+
+test:
+  commands:
+    - echo
```

Comment:
can you improve the test, for example by useing something like: https://github.com/conda-forge/zlib-feedstock/blob/master/recipe/meta.yaml#L27